src/main/java/uk/gov/pay/publicauth/model/CreateTokenRequest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.pay.publicauth.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+
+import static java.util.UUID.randomUUID;
+import static uk.gov.pay.publicauth.model.TokenPaymentType.CARD;
+import static uk.gov.pay.publicauth.model.TokenSource.API;
+
+public class CreateTokenRequest {
+
+    @NotNull private final String accountId;
+    @NotNull private final String description;
+    @NotNull private final String createdBy;
+    private final TokenPaymentType tokenPaymentType;
+    private final TokenSource tokenSource;
+    private final TokenLink tokenLink = TokenLink.of(randomUUID().toString());
+
+
+
```

Comment:
a few blank lines that could be removed

Reply:
👍 thanks, just removing, if you could re-approve please when its ready.